projects/canopy/src/lib/forms/date/docs/guide.stories.mdx

```diff
@@ -47,130 +54,127 @@ and in your HTML:
 
 The current significance of errors with their recommended messages is as follows.
 
 #### 1. Invalid fields
 
-One field is invalid, this may be a non numerical character, or a number outside the valid range .e.g 13 for a month.
-\`Enter a valid month\`
+One field is invalid, this may be a non-numerical character, or a number outside the valid range .e.g 13 for a month.
+`Enter a valid month`
 
-~~~json
+```json
 
 { invalidField: 'month'}
 
-~~~
+```
 
 
 As above but with two fields displaying an error.
-\`Enter a valid month and year\`
+`Enter a valid month and year`
 
-~~~json
+```json
 
 { invalidFields: ['day', 'month']}
 
-~~~
+```
 
 If all three fields are invalid a generic 'invalidDate' message is provided.
-\`Enter a valid date of birth\`
+`Enter a valid date of birth`
 
-~~~json
+```json
 
 { invalidDate: true }
 
-~~~
+```
 
 #### 2. Required fields
 
 If one field is empty either through form submission or inline deletion of the content.
-\`Date of birth must include month\`
+`Date of birth must include month`
 
-~~~json
+```json
 
 { requiredField: 'month'}
 
-~~~
+```
 
 As above but with two fields missing.
-\`Date of birth must include a month and year\`
+`Date of birth must include a month and year`
 
 
-~~~json
+```json
 
 { invalidFields: ['day', 'month']}
 
-~~~
+```
 
 If all three fields are missing an 'invalidDate' message is provided.
-\`Enter a valid date of birth\`
+`Enter a valid date of birth`
 
-~~~json
+```json
 
 { invalidDate: true }
 
-~~~
+```
 
 #### 2. Invalid date
 
-If all of the individual fields are correct but they do not concatenate to a valid date. e.g. 30 02 2020
+If all the individual fields are correct, but they do not concatenate to a valid date. e.g. 30 02 2020. `Enter a valid date of birth`
 
-~~~json
+```json
 
 { invalidDate: true }
 
-~~~
-\`Enter a valid date of birth\`
+```
 
 ### External validation
 
 It is also possible to provide your own custom validation messages using <a href="https://angular.io/guide/form-validation#custom-validators" target="_blank">custom validators</a>
-and the validation component. This is particularly useful for checking things like wether the date is in the past or the future. Under the hood the date field uses the <a href="https://date-fns.org/" target="_blank">date-fns</a> library as a peer dependency, to keep build size to a minimum it may be worth considering using the same library in your application.
+and the validation component. This is particularly useful for checking things like whether the date is in the past or the future. Under the hood the date field uses the <a href="https://date-fns.org/" target="_blank">date-fns</a> library as a peer dependency, to keep build size to a minimum it may be worth considering using the same library in your application.
 
-~~~js
+```js
```

Comment:
As per a previous file, this is TS, not js. I'm not sure if it supports TS though

Reply:
ts seem to be working ok - not sure if it's adding any more highlight though